cellprofiler/module.py

```diff
@@ -938,7 +938,8 @@ def run(self, workspace):
         y = cellprofiler.image.Image(
             dimensions=dimensions,
             image=y_data,
-            parent_image=x
+            parent_image=x,
+            convert=False
```

Comment:
Resolves #2687. `convert=True` caused an error in modules downstream of Erosion due to datatype.